### PR TITLE
[acts] Add version 8.2 and support master -> main branch naming switch

### PR DESCRIPTION
**Pull request by HadrienG2**
I think the PR title says it all. Feedback on the way in which I handled the development branch name change is welcome.

### CODE REVIEW

**Review by michaelkuhn — DISMISSED** [review]
LGTM (some other packages are handling the master rename the same way)

File: var/spack/repos/builtin/packages/acts/package.py [hunk 1]
```diff
@@ -36,7 +36,9 @@ class Acts(CMakePackage, CudaPackage):
     tags = ['hep']
 
     # Supported Acts versions
-    version('master', branch='master')
+    version('main', branch='main')
+    version('master', branch='main')  # Alias kept for script compatibility
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/acts/package.py]: Would it work if you add `deprecated=True`? That has the benefit of not showing the alias in `spack info`

Comment (HadrienG2) [reply var/spack/repos/builtin/packages/acts/package.py]: I wouldn't mind myself as I migrated all my scripts to `main`, just kept the `master` alias for the benefit of not suddenly breaking other people's scripts.

Comment (alalazo) [reply var/spack/repos/builtin/packages/acts/package.py]: Sure, with `deprecated=True` scripts would not be broken but people looking at the available versions won't see `master`.

Comment (HadrienG2) [reply var/spack/repos/builtin/packages/acts/package.py]: Thanks for the clarification, I implemented the suggested change :)

**Review by alalazo — APPROVED** [review]

**Pull request merged**